app/Controllers/RedirectController.php

```diff
@@ -11,6 +12,13 @@ static public function redirect() {
         if (isset($_GET['type'], $_GET['term'])) {
             $term = trim($_GET['term']);
             switch ($_GET['type']) {
+                case 'url':
+                    $endpoint = to_endpoint($term);
+                    if (!$endpoint) {
+                        ErrorHandler::show('Invalid or unknown TikTok URL format');
```

Comment:
ErrorHandler::show only accepts [Meta](https://github.com/pablouser1/TikScraperPHP/blob/master/src/Models/Meta.php) objects, I will make another method that accepts strings, but for now let's just echo the error.
```suggestion
                        echo 'Invalid or unknown TikTok URL format';
```